controls/nist_ocp4.yml

```diff
@@ -13880,12 +13905,26 @@ controls:
   levels:
   - high
 - id: SC-12(2)
-  status: pending
+  status: automated
   notes: |-
-    A complete control response is planned. Engineering progress can be
-    tracked via:
+    FIPS mode can be enabled in OpenShift through a flag that
+    can be set at installation time [1]. Follow the relevant
+    documentation for the applicable cloud provider [2][3][4][5]
+    for more information. But, note that this is also applicable
+    in on-prem deployments.
+
+    Managing keys for external components is an organizational
+    procedure outside the scope of OpenShift configuration.
+
+    For more details about requesting and managing the certificates
+    served by the service-ca controller, refer to [5].
+
+    [1] https://docs.openshift.com/container-platform/latest/installing/installing-fips.html
+    [2] https://docs.openshift.com/container-platform/latest/installing/installing_aws/installing-aws-government-region.html#installation-configuration-parameters_installing-aws-government-region
+    [3] https://docs.openshift.com/container-platform/4.7/installing/installing_azure/installing-azure-customizations.html
+    [4] https://docs.openshift.com/container-platform/4.7/installing/installing_gcp/installing-gcp-customizations.html
+    [5] https://docs.openshift.com/container-platform/latest/security/certificates/service-serving-certificate.html
 
-    https://issues.redhat.com/browse/CMP-498
   rules: []
```

Comment:
Seems to me like the rule is missing from here. We do have a rule that could be used.